website/_dogfooding/_pages tests/code-block-tests.mdx

```diff
@@ -27,6 +27,21 @@ See:
 - https://github.com/facebook/docusaurus/pull/3749
 - https://github.com/facebook/docusaurus/pull/6177
 
+## Code block with/without the good prism language case in additionalLanguages[] tests
+
+```php
```

Comment:
that would be nice to add 2 other cases like `PHP` and `PhP` there to ensure we also support case insensitive language metastring

Reply:
Well, i already tried doing that before making a pr.
But only php code block works and other like you mentioned doesn't highlight code

Reply:
Well, I have added them and I also saw that the VSCode Markdown preview supports highlighting all these blocks. So, maybe we should do it too.